### ISSUE TITLE
Citing patent filter

### ISSUE THREAD

**Issue by zilch42** [cambialens/lens-api-doc#4]
Hi,   
I'd like to be able to filter scholarly articles returned by the API based on the year of patent filing.   For example, I'm currently interested in getting back all of the articles published by Australian researchers (published any time) that were cited by patents filed between 2014-2018.   

If that can be done already, please let me know. But I haven't been able to find it. If not, I'd like to   put it in as a feature request.    

Cheers

**Comment by rosharma9** [comment]
Hi @zilch42 , This feature is not yet available through APIs as we are working on Patent APIs. We would definitely consider it as a feature request there are similar requests from other users too.
However for now, you can use an alternative solution using our UI to export the patent records and use it to query against Scholarly API.
For your search criteria, you can [export results from UI](https://link.lens.org/wyU8zjyi8rd) in your desired format and extract the Lens IDs. You can then use those IDs to [create a terms query](https://docs.api.lens.org/examples.html#get-the-metadata-for-scholarly-works-that-are-cited-by-a-list-of-patents-using-the-patent-lens_id) to access scholarly articles through the APIs. 
I hope that helped.

Cheers

**Comment by zilch42** [comment]
Hi @rosharma9 , Thanks for that.

I tried going through the UI a few days ago, and requested the dataset be emailed to me when ready as it was about 39k records. I haven't seen that email come through yet though. I imagine after 2-3 days it's probably not coming?
I also tried going through PatCite yesterday using the patent lens ids I got from my scholarly API query but the export file I got back from that was empty. I contacted support about that yesterday. I'm out of ideas of other things to try. 

Cheers
